.circleci/config.yml

```diff
@@ -1,5 +1,8 @@
 version: 2.1
-
+parameters:
+  run_default_workflows:
+    default: true
+    type: boolean
```

Comment:
Does it make sense to have this when the same flag affects both workflows?

Reply:
Yeah, was unsure about that - this is just handy for debugging, if you set this to false,
and then introduce another parameter like this one that is defaulting to true for the workflow you want to run. But alone this has no direct use.. Would you remove this completely, or rather introduce a "dummy" test workflow that makes use of the above strategy but is set to false?

Reply:
I think it could make sense to have one flag for each of the default workflows.
Did you test if you can trigger the nightly workflow via the API while the cron trigger is set?

Reply:
No, this didn't seem to work, so the `when` seems to work like an initial filter, but not an event trigger.

Reply:
Ok... Well I guess we could add a copy of the nightly workflow that defaults to false.
But let's not keep delaying this PR for that.